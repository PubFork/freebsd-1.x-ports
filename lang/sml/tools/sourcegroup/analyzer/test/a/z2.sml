functor F (A:A) = struct
  val c = C.z
  local open A in
    val b = B.x
   end
  val d = D.y
end

(*
  {efu F [isi A dst A:A ist C
          {Ost A [ist B]} ist D]}
*)
