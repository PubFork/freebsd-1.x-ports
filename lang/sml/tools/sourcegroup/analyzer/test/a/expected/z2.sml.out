source sml "a/z2.sml"
  import structure B C D 
  import signature A 
  export functor F ;

